src/erl.rs

```diff
@@ -538,6 +538,12 @@ fn bare_clause_guard(guard: &TypedClauseGuard, env: &mut Env) -> Document {
 
         ClauseGuard::Float { value, .. } => value.to_string().to_doc(),
 
+        // TODO: .to_string() is not implemented?
+        ClauseGuard::Tuple { elems, ..} => elems
+            .map(|&x| x.to_string())
+            .collect::<Vec<&str>>()
+            .join(",").to_doc(),
```

Comment:
This doesn't render correct Erlang syntax and it won't format it correctly. We want to use the pretty printing algebra rather than string formatting. For a reference check out how TypedExpr::Tuple and Pattern::Tuple are printed in this module.